File: Taller Tipos de clases/C#/Clase Normal/Carro.cs
class Carro
{
    public string Marca;
    public int Modelo;

    public Carro()
    {
        Marca = "NA";
        Modelo = 0;
    }

    public Carro(string marca, int modelo)
    {
        Marca = marca;
        Modelo = modelo;
    }

    public Carro(string marca)
    {
        Marca = marca;
        Modelo = 2024;
    }

    public Carro(Carro c)
    {
        Marca = c.Marca;
        Modelo = c.Modelo;
    }
}

File: Taller Tipos de clases/JAVA/Clase Normal/Carro.java
class Carro {

    String marca;
    int modelo;

    // Constructor sin parámetros
    public Carro() {
        marca = "NO";
        modelo = 0;
    }

    // Constructor con parámetros
    public Carro(String marca, int modelo) {
        this.marca = marca;
        this.modelo = modelo;
    }

    // Constructor sobrecargado
    public Carro(String marca) {
        this.marca = marca;
        this.modelo = 2025;
    }

    // Constructor copia
    public Carro(Carro c) {
        this.marca = c.marca;
        this.modelo = c.modelo;
    }
}
